charts/newrelic-logging/templates/configmap.yaml

```diff
@@ -12,14 +12,23 @@ data:
     {{- if .Values.fluentBit.config.inputs }}
       {{- .Values.fluentBit.config.inputs | nindent 4 }}
     {{- end }}
+    {{- if .Values.fluentBit.config.extraInputs }}
+      {{ .Values.fluentBit.config.extraInputs | indent 4}}
```

Comment:
Hi @els-ipatel , I tested the Helm chart installation with extraInputs/extraFilters/extraOutputs and found that this `indent` needs to be a `nindent` instead (like the rest in this file). Otherwise, the resulting Fluent Bit configuration has an invalid indentation. Could you please correct this? Thank you.

Reply:
Also please note that a `-` is missing after the `{{-` in these lines for the resulting indentation to be correct. Here is how it should look like everywhere:
```
{{- .Values.fluentBit.config.extraInputs | indent 4 }}
```

Reply:
fixed.